src/scaraplate/__main__.py

```diff
@@ -1,8 +1,26 @@
+import collections
+from typing import Mapping
+
 import click
 
 from scaraplate.rollup import rollup as _rollup
 
 
+def validate_extra_context(ctx, param, value):
```

Comment:
Technically, there is nothing about validation, but it's about value conversion.

Reply:
I don't like the name either, but I decided to keep it in order to not divert from cookiecutter from which this function was vendored.